src/classes/Recipe.js

```diff
@@ -1,15 +1,30 @@
 class Recipe {
   constructor(recipeInfo) {
-    this.recipeNameInfo = recipeInfo.newIngredients;
     this.id = recipeInfo.id;
     this.image = recipeInfo.image;
     this.ingredients = recipeInfo.ingredients;
     this.instructions = recipeInfo.instructions;
     this.name = recipeInfo.name;
     this.tags = recipeInfo.tags;
     this.ingredientsNeeded = [];
+    this.ingredientsDetails = [];
   }
```

Comment:
added a method to grab an object containing the ingredient id, name, quantity,  and cost per unit and push them into this.ingredientsDetails